react/__tests__/__snapshots__/Backdrop.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Backdrop /> should match snapshot 1`] = `[Function]`;
```

Comment:
something is wrong

Reply:
I talked with @iaronaraujo and @victorhmp and we thought about to remove the snapshot tests, what do you think?